src/commands/ext-install.ts

```diff
@@ -83,6 +84,12 @@ export default new Command("ext:install [extensionName]")
     }
     let source;
     let extVersion;
+
+    // TODO(b/220900194): Track url install usage to determine if we can remove this feature.
+    if (isUrlPath(extensionName)) {
+      track("Extension Install", "Install by url path", options.interactive ? 1 : 0);
```

Comment:
void track()?

Reply:
ah good catch, I made changes for this file in the launch branch, good idea to keep this clean